releasenotes/notes/feature-inject-trace-lambda-invoke-818866a5de2015a9.yaml

```diff
@@ -0,0 +1,65 @@
+---
+prelude: >
+    Replace this text with content to appear at the top of the section for this
+    release. All of the prelude content is merged together and then rendered
+    separately from the items listed in other parts of the file, so the text
+    needs to be worded so that both the prelude and the other items make sense
+    when read independently. This may mean repeating some details. Not every
+    release note requires a prelude. Usually only notes describing major
+    features or adding release theme details should have a prelude.
+features:
+  - |
```

Comment:
You just need to fill in this section here 🙂

Reply:
Oops! Added

Reply:
great! thanks